tf2_ros/doc/index.rst

```diff
@@ -0,0 +1,80 @@
+Overview
+========
+
+This package contains the ROS 2 bindings for the tf2 library, for both Python and C++.
+
+1.1 Broadcasting Transforms
+---------------------------
+  * :class:`tf2_ros::TransformBroadcaster()` constructor of
+    (:ref:`exhale_class_classtf2__ros_1_1_transform_broadcaster`)
+  * :class:`tf2_ros::TransformBroadcaster::sendTransform` to send transforms
+
+Similarly static transforms can be sent by:
+
+  * :class:`tf2_ros::StaticTransformBroadcaster()`, constructor of
+    (:ref:`exhale_class_classtf2__ros_1_1_static_transform_broadcaster`)
+  * :class:`tf2_ros::StaticTransformBroadcaster::sendTransform` to send static transforms
+
+1.2 Using Published Transforms
+------------------------------
+For most purposes using tf2_ros will be done using tf2_ros::Buffer (:ref:`exhale_class_classtf2__ros_1_1_buffer`). It's main public API is defined by tf2_ros::BufferInterface (:ref:`exhale_class_classtf2__ros_1_1_buffer_interface`). Typically it will be populated using a tf2_ros::TransformListener (:ref:`exhale_class_classtf2__ros_1_1_transform_listener`) which subscribes to the appropriate topics.
```

Comment:
```suggestion
For most purposes using tf2_ros will be done using tf2_ros::Buffer (:ref:`exhale_class_classtf2__ros_1_1Buffer`). It's main public API is defined by tf2_ros::BufferInterface (:ref:`exhale_class_classtf2__ros_1_1BufferInterface`). Typically it will be populated using a tf2_ros::TransformListener (:ref:`exhale_class_classtf2__ros_1_1TransformListener`) which subscribes to the appropriate topics.
```